custom_components/tahoma/climate_devices/atlantic_pass_apc_heating_zone.py

```diff
@@ -0,0 +1,334 @@
+"""Support for Atlantic Pass APC Heating Zone."""
+import logging
+from typing import List, Optional
+
+from homeassistant.components.climate import (
+    SUPPORT_PRESET_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+    ClimateEntity,
+)
+from homeassistant.components.climate.const import (
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    PRESET_AWAY,
+    PRESET_COMFORT,
+    PRESET_ECO,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    EVENT_HOMEASSISTANT_START,
+    STATE_UNAVAILABLE,
+    STATE_UNKNOWN,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.event import async_track_state_change
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+CUSTOM_PRESET_DEROGATION = "Derogation"
+CUSTOM_PRESET_AUTO = "Auto"
+CUSTOM_PRESET_STOP = "Stop"
+
+COMMAND_REFRESH_COMFORT_HEATING_TARGET_TEMPERATURE = (
```

Comment:
Can you please add all these states and commands to https://github.com/iMicknl/python-overkiz-api/tree/main/pyoverkiz/enums ?

Reply:
Should i also add commands / states not used in this class ?

Reply:
Only add the ones you need